lxc/file.go

```diff
@@ -264,6 +265,12 @@ func (c *cmdFilePull) Run(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		targetIsDir = true
+	} else if len(targetDirs) > 1 && c.file.flagMkdir {
+		dirs := strings.Join(targetDirs[:len(targetDirs)-1], string(os.PathSeparator))
+		err := os.MkdirAll(dirs, 0755)
```

Comment:
@masnax feels like the 0755 should be moved to a constant and then used here and line 263 to avoid any accidental changes in the future where one creation mode is changed without the changing other.